crates/typst/src/text/smallcaps.rs

```diff
@@ -23,10 +25,15 @@ use crate::text::TextElem;
 /// = Introduction
 /// #lorem(40)
 /// ```
-#[func(title = "Small Capitals")]
-pub fn smallcaps(
-    /// The text to display to small capitals.
-    body: Content,
-) -> Content {
-    body.styled(TextElem::set_smallcaps(true))
+#[elem(name = "smallcaps", title = "Small Capitals", Show)]
+pub struct SmallCapsElem {
+    #[required]
+    pub body: Content,
```

Comment:
Can you add a brief documentation comment to this?

Reply:
Thanks for your reply, I have added the test & example and adjusted the docs.